Packages/vcs/vcs/VTKPlots.py

```diff
@@ -315,7 +313,7 @@ def configureEvent(self, obj, ev):
             restart_anim = self.canvas.configurator.animation_timer is not None
         else:
             restart_anim = False
-        self.canvas.clear(render=False)
+        self.canvas.clear(render=False, preserve_display=True)
```

Comment:
Can you add a line describing preserve_display in the documentation for clear? Is there a place where all parameters for clear() are listed and described?